lumicks/pylake/correlated_stack.py

```diff
@@ -129,36 +224,48 @@ def from_data(cls, data, name=None, start_idx=0, stop_idx=None):
         new_correlated_stack.stop_idx = (new_correlated_stack.src.num_frames if stop_idx is None else stop_idx)
         return new_correlated_stack
 
-    def plot(self, frame=0, **kwargs):
+    def plot(self, frame=0, channel="rgb", show_title=True, **kwargs):
         """Plot image from image stack
 
         Parameters
         ----------
         frame : int, optional
             Index of the frame to plot.
+        channel : 'rgb', 'red', 'green', 'blue', None; optional
+            Channel to plot for RGB images (None defaults to 'rgb')
+            Not used for grayscale images
+        show_title : bool, optional
+            Controls display of auto-generated plot title
         **kwargs
             Forwarded to :func:`matplotlib.pyplot.imshow`.
         """
         import matplotlib.pyplot as plt
 
         default_kwargs = dict(
-            cmap='gray'
+            cmap='gray',
+            vmax=None
         )
+        kwargs = {**default_kwargs, **kwargs}
 
-        image = self._get_frame(frame).data
-        plt.imshow(image, **{**default_kwargs, **kwargs})
+        fr = self._get_frame(frame)
+        image = fr._get_plot_data(channel, vmax=kwargs["vmax"])
+        if isinstance(kwargs["vmax"], str):
+            kwargs.pop("vmax")
+        plt.imshow(image, **kwargs)
 
-        if self.num_frames == 1:
-            plt.title(self.name)
-        else:
-            plt.title(f"{self.name} [frame {frame}/{self.num_frames}]")
+        if show_title:
+            if self.num_frames == 1:
+                plt.title(self.name)
+            else:
+                # display with 1-based index for frames and total frames
+                plt.title(f"{self.name} [frame {frame+1}/{self.num_frames}]")
```

Comment:
why optional?

Reply:
While I was testing, it was easiest to plot a grid of images using different conditions for the vmax options and the titles kept getting in the way. Rather than setting the title to an empty string each time, it seemed useful to be able to turn the auto-generated title off. It's a small change and the default is still the same as before; but we can get rid of the option if you think that's best